dpcpp/CMakeLists.txt

```diff
@@ -43,6 +46,10 @@ target_sources(ginkgo_dpcpp
 ginkgo_compile_features(ginkgo_dpcpp)
 
 set(GINKGO_DPCPP_FLAGS ${GINKGO_COMPILER_FLAGS} -fsycl)
+if (GINKGO_DPCPP_VERSION AND GINKGO_DPCPP_VERSION VERSION_GREATER_EQUAL 20201005)
```

Comment:
I think we will only support the first stable version not these beta afterwards?
maybe less than the version, throwing the error is enough.

Reply:
Since for now there are still the older versions on the devcloud, I thought maybe you would use them somehow. Since the flag didn't exist then, I'm not propagating it. That allows using any older version without having to apply a manual patch or something.

Reply:
In the end I moved everything to only use beta10. I also changed the math implementation. We still need to use `cl::sycl::sqrt` instead of plain `sqrt` AFAIK.